glib/src/char.rs

```diff
@@ -169,10 +169,10 @@ mod tests {
 
     #[test]
     fn converts_single_byte_chars() {
-        assert_eq!(Char::try_from(0 as char), Ok(Char(0_i8)));
-        assert_eq!(UChar::try_from(0 as char), Ok(UChar(0_u8)));
-        assert_eq!(UChar::try_from(255 as char), Ok(UChar(255_u8)));
-        assert_eq!(UChar::try_from('ñ'), Ok(UChar(241_u8)));
+        assert_eq!(Char::try_from(0 as char), Ok(Char(0 as c_char)));
+        assert_eq!(UChar::try_from(0 as char), Ok(UChar(0 as c_uchar)));
+        assert_eq!(UChar::try_from(255 as char), Ok(UChar(255 as c_uchar)));
+        assert_eq!(UChar::try_from('ñ'), Ok(UChar(241 as c_uchar)));
```

Comment:
Maybe use `as _`, it might not complain about it.

Reply:
No:

```
warning: casting integer literal to `i8` is unnecessary
   --> glib/src/char.rs:172:55
    |
172 |         assert_eq!(Char::try_from(0 as char), Ok(Char(0 as _)));
    |                                                       ^^^^^^ help: try: `0_i8`
    |
```

Reply:
Silenced the warnings for now until someone has a better idea. The same would've also happened in the non-test code on ARM (or wherever a C `char` is unsigned).

Reply:
I guess it's fine.